examples/tess/beta_proposal.jl

```diff
@@ -0,0 +1,214 @@
+end
+
+function make_proposal_dist_multidim_beta(pop::abc_population_type, tau_factor::Float64; verbose::Bool = false)
+    make_proposal_dist_multidim_beta(pop.theta, pop.weights, tau_factor, verbose=verbose)
+end
+
+function make_proposal_dist_multidim_beta_dirichlet(theta::AbstractArray{Float64,2}, weights::AbstractArray{Float64,1},  tau_factor::Float64; verbose::Bool = false)
+    global sim_param_closure
+    local limitP::Array{Float64,1} = get_any(EvalSysSimModel.sim_param_closure, "p_lim_arr", Array{Float64,1})
+    local p_dim = length(limitP)-1
+    local r_dim = length(get_any(EvalSysSimModel.sim_param_closure, "r_lim_arr", Array{Float64,1}))-1
+    
+    theta_mean =  sum(theta.*weights',dims=2) # weighted mean for parameters
+    theta_var = ABC.var_weighted(theta'.-theta_mean',weights)  # scaled, weighted covar for parameters
+    tau_factor_indiv = fill(tau_factor,length(theta_var))
+
+    dist_arr = ContinuousDistribution[]
+    for j in 1:p_dim
+        max_col_rate = 3*log(limitP[j+1]/limitP[j])/log(2)
+        col_startidx = (j-1)*(r_dim+1)+1
+        #tau_factor_indiv[col_startidx] = 2.0
+
+        # if verbose
+        #     println("total: ",theta_mean[col_startidx]," ",theta_var[col_startidx])
+        # end
+        # for i in (col_startidx+1):(col_startidx+r_dim)
+        #     mean_ratio = sum(theta[col_startidx,:].*theta[i,:].*weights) /(theta_mean[col_startidx]*theta_mean[i]) # weighted mean for parameters
+        #     var_ratio = var_weighted(vec(theta[col_startidx,:].*theta[i,:]).-(theta_mean[col_startidx]*theta_mean[i]),weights)/(2 * theta_mean[col_startidx] * theta_var[i]) # scaled, weighted covar for parameters
+        #     if verbose
+        #         println("i=",i,": ",theta_mean[i]," ",theta_var[i]," ratios: ",mean_ratio, " ",var_ratio)
+        #     end
+        #     var_ratio  = var_ratio  >= one(var_ratio)  ? var_ratio  : one(var_ratio)
+        #     tau_factor_indiv[i] = tau_factor*var_ratio
+        # end
+        # if verbose
+        #     flush(stdout)
+        # end
+
+        #=
+        println("mean= ",theta_mean)
+        println("var= ",theta_var)
+        println("tau= ",tau_factor_indiv)
+        for i in 1:length(theta_mean)
+            println("a= ",alpha(theta_mean[i],tau_factor*theta_var[i]), "  b= ",beta(theta_mean[i],tau_factor*theta_var[i]))
+        end
+        =#
+        
+        dist_arr = vcat(dist_arr, make_beta_transformed(theta[col_startidx,:], weights, xmin=0.0, xmax=max_col_rate, xmean=theta_mean[col_startidx]/max_col_rate, xvar=theta_var[col_startidx]/max_col_rate^2, tau_factor=tau_factor_indiv[col_startidx]), ContinuousDistribution[ make_beta(theta[i,:], weights, xmean=theta_mean[i], xvar=theta_var[i], tau_factor=tau_factor_indiv[i]) for i in (col_startidx+1):(col_startidx+r_dim)])
+    end
+
+dist = CompositeDist(dist_arr)
+end
+
+function make_proposal_dist_multidim_beta_dirichlet(pop::abc_population_type, tau_factor::Float64; verbose::Bool = false)
```

Comment:
Is this file basically a copy & paste from the other directory?  If so, then maybe I should move much of this into the core SysSim code (i.e., into src directory).